integration_test/Cargo.toml

```
@@ -5,6 +5,6 @@ authors = ["Steven Roose <steven@stevenroose.org>"]
 
 [dependencies]
 bitcoincore-rpc = { path = "../client" }
-bitcoin = { version = "0.26", features = [ "use-serde", "rand" ] }
+bitcoin = { version = "0.27", features = [ "use-serde", "rand" ] }
```

Currently `master` is at 0.28.0, but I believe we should drop the patch specification as before so users of the library can resolve the latest patch.

So just a rebase is in order and then we can drop the patch version :)

```suggestion
bitcoin = { version = "0.28", features = [ "use-serde", "rand" ] }
```